src/Components/Landing/Landing.module.scss

```diff
@@ -0,0 +1,196 @@
+@mixin flexDisplay($position, $direction) {
+  display: flex;
+  flex-direction: $direction;
+  align-items: center;
+  justify-content: $position;
+}
+
+//Main style file starts here
+
+.header {
+  display: flex;
```

Comment:
Add a `justify-content: center` to center the contents inside the flexbox.

Reply:
Added justify-content: center